content/learning-paths/smartphones-and-mobile/using-neon-intrinsics-to-optimize-unity-on-android/8-best-practice.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Profiling best practice
+weight: 9
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+1. Try to capture representative and repeatable captures
```

Comment:
this feels like it needs a heading, which is probably "Profiling best practice" even though that's the page heading as well. But it looks bald without it.

Reply:
it looked weird repeating the header, and looked worse if i used different wording.
i have added a line to introduce the list - i think this looks better.
done

Reply:
i've added a title "capturing reliable performance data" - i think it follows on nicely from the "profiling best practice" page title?